packages/server/src/service/static/pug/org.pug

```diff
@@ -63,10 +63,18 @@ block content
 
     ul.mt-2.list-group
       each info in stateInfos
+        - var namePart = info.name
```

Comment:
Don't rewrite the logic in the template, since `name` will now be backward compatible!